utils/sv_easing_functions.py

```diff
@@ -328,35 +328,46 @@ def BounceEaseInOut(p, settings=defaultBounceSettings):
 
 
 easing_dict = {
+    # LINEAR
```

Comment:
if the easing dict had tuples as keys (which i'm OK with) you could sort on first or second item in the key-tuple
 
f.ex:

```python
easing_dict = {
    (0, 0): LinearInterpolation,
    (1, 4): QuadraticEaseIn,
    (2, 5): QuadraticEaseOut,
    (3, 6): QuadraticEaseInOut,
    (4, 7): CubicEaseIn,
    (5, 8): CubicEaseOut,
    (6, 9): CubicEaseInOut,
    (7, 10): QuarticEaseIn,
    (8, 11): QuarticEaseOut,
    (9, 12): QuarticEaseInOut,
   ...
```
but...uch..